vm/src/obj/objbytes.rs

```diff
@@ -35,23 +37,29 @@ use std::str::FromStr;
 /// - an integer";
 #[pyclass(name = "bytes")]
 #[derive(Clone, Debug)]
-pub struct PyBytes {
```

Comment:
Making `PyBytes` and `PyByteArray`  `pub(crate)` does seem like a good idea, providing that `PyBytesLike` remains `pub`, so that people can specify that they want a function to support any sort of bytes.